src/System.ServiceModel.Duplex/tests/System.ServiceModel.Duplex.Tests.builds

```diff
@@ -6,6 +6,10 @@
       <TargetGroup>netstandard1.3</TargetGroup>
       <TestTFMs>netcore50;netcoreapp1.1</TestTFMs>
```

Comment:
We should run the netstandard1.3 build of the tests against netcoreapp2.0 as well.

It's up to WCF if they would prefer to build their tests against ns2.0 and test that as well.  Typically I'd only expect this to matter if the tests *need* ns2.0 surface area, either from CoreFx or WCF libs, or if they want to test assembly loading in both scenarios since in the ns1.x case the test will have all System.* references but in the ns2.x case the test will have a netstandard.dll reference.

I think its important to have at least one test building against ns2.0 since that'll test consumption of WCF libs in ns2.0 projects.